lib/desired-caps.js

```diff
@@ -383,7 +383,11 @@ const desiredCapConstraints = /** @type {const} */ ({
   pageLoadStrategy: {
     isString: true,
     inclusionCaseInsensitive: ['none', 'eager', 'normal']
-  }
+  },
+  enableWdaRestart: {
```

Comment:
the capability name is too broad. I would rather use something like `restartWdaOnCrash`